core/id/internal_id_wrappers.go

```diff
@@ -0,0 +1,441 @@
+func (id InternalSequence) SchemaName() string {
+	return Internal(id).Segment(0)
+}
+
+// SequenceName returns the name of the sequence.
+func (id InternalSequence) SequenceName() string {
+	return Internal(id).Segment(1)
+}
+
+// SchemaName returns the schema name of the table.
+func (id InternalTable) SchemaName() string {
+	return Internal(id).Segment(0)
+}
+
+// TableName returns the table's name.
+func (id InternalTable) TableName() string {
+	return Internal(id).Segment(1)
+}
+
+// SchemaName returns the schema name of the type.
+func (id InternalType) SchemaName() string {
+	return Internal(id).Segment(0)
+}
+
+// TypeName returns the type's name.
+func (id InternalType) TypeName() string {
+	return Internal(id).Segment(1)
+}
+
+// SchemaName returns the schema name of the view.
+func (id InternalView) SchemaName() string {
+	return Internal(id).Segment(0)
+}
+
+// ViewName returns the view's name.
+func (id InternalView) ViewName() string {
+	return Internal(id).Segment(1)
+}
+
+// IsValid returns whether the ID is valid.
+func (id InternalAccessMethod) IsValid() bool { return Internal(id).IsValid() }
+
+// IsValid returns whether the ID is valid.
+func (id InternalCheck) IsValid() bool { return Internal(id).IsValid() }
+
+// IsValid returns whether the ID is valid.
+func (id InternalCollation) IsValid() bool { return Internal(id).IsValid() }
+
+// IsValid returns whether the ID is valid.
+func (id InternalColumnDefault) IsValid() bool { return Internal(id).IsValid() }
+
+// IsValid returns whether the ID is valid.
+func (id InternalDatabase) IsValid() bool { return Internal(id).IsValid() }
+
+// IsValid returns whether the ID is valid.
+func (id InternalEnumLabel) IsValid() bool { return Internal(id).IsValid() }
+
+// IsValid returns whether the ID is valid.
+func (id InternalForeignKey) IsValid() bool { return Internal(id).IsValid() }
+
+// IsValid returns whether the ID is valid.
+func (id InternalFunction) IsValid() bool { return Internal(id).IsValid() }
+
+// IsValid returns whether the ID is valid.
+func (id InternalIndex) IsValid() bool { return Internal(id).IsValid() }
+
+// IsValid returns whether the ID is valid.
+func (id InternalNamespace) IsValid() bool { return Internal(id).IsValid() }
+
+// IsValid returns whether the ID is valid.
+func (id InternalOID) IsValid() bool { return Internal(id).IsValid() }
+
+// IsValid returns whether the ID is valid.
+func (id InternalSequence) IsValid() bool { return Internal(id).IsValid() }
+
+// IsValid returns whether the ID is valid.
+func (id InternalTable) IsValid() bool { return Internal(id).IsValid() }
+
+// IsValid returns whether the ID is valid.
+func (id InternalType) IsValid() bool { return Internal(id).IsValid() }
+
+// IsValid returns whether the ID is valid.
+func (id InternalView) IsValid() bool { return Internal(id).IsValid() }
+
+// Internal returns the unwrapped ID.
+func (id InternalAccessMethod) Internal() Internal { return Internal(id) }
```

Comment:
These calls are all kind of confusing in context, maybe just because of the name, and don't really add much. Maybe inline all of them. Or if you want to keep them, combine with the name change of `Internal` to `Id` and call these `ToId` or `AsId`